systems/plants/ContactForceTorqueSensor.m

```diff
@@ -123,7 +123,7 @@
         nD = length(tsmanip.LCP_cache.data.contact_data.d);
         nC_body = nA + nB;
         nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
-        nL = manip.getNumJointLimitConstraints;
+        nL = sum(tsmanip.LCP_cache.data.possible_limit_indices);
         nP = 2*manip.num_position_constraints;  % number of position constraints
```

Comment:
Actually -- should the 2x multiplier still be here? It does not appear in the production of nP over in the solve LCP function in the TSRBM, and it's been causing trouble for me as I'm trying to move this code over to apply for the FT sensor on the arm.